lib/email_reply_parser.rb

```diff
@@ -141,6 +173,24 @@ def read(text)
       SIG_REGEX = Regexp.new(SIGNATURE)
     end
 
+    # Regular expression for regards
+    #
+    # Returns a Regexp instance if regards are configured, otherwise it returns
+    # nil
+    def regards_regex
+      return nil if EmailReplyParser.configuration.regards.empty?
+      value = EmailReplyParser.configuration.regards.map do |regard|
+        "(#{regard.reverse}$)"
+      end.join('|')
+
+      begin
+        require 're2'
+        RE2::Regexp.new(value, case_sensitive: false)
+      rescue LoadError
+        Regexp.new(value, ignore_case: true)
```

Comment:
Currently, we'll create a new `Regexp` object with each email message we parse, though the content of the regular expression won't be changing. I think we want to memoize it in one way or another.